sw/device/lib/dif/dif_kmac.h

```diff
@@ -155,6 +155,12 @@ typedef struct dif_kmac_config {
    * granularity.
    */
   bool output_big_endian;
+
+  /**
+   * Place kmac inside key sideload mode
```

Comment:
Did you mean `into` instead of `inside`?

Reply:
indeed i meant into.  Let me update that.